lib/screens/home/pages/members/bloc/members_page_bloc.dart

```diff
@@ -25,47 +26,67 @@ class MembersPageBloc extends Bloc<MembersPageEvent, MembersPageState> {
     }
   }
 
-  Stream<MembersPageState> _mapEventToMembersShowed(MembersPageEvent event) async* {
+  Stream<MembersPageState> _mapEventToMembersShowed(
+      MembersPageEvent event) async* {
     final currentState = state;
 
     if (event is MembersPageShowed && !_hasReachedMax(currentState)) {
       try {
         if (currentState is MembersPageInitial) {
           yield MembersPageLoading();
-          final List<User> users = await userRepository.getVerifiedUsers(pageNumber);
-          yield MembersPageSuccess(users: users, hasReachedMax: false);
+          final List<User> users =
+              await userRepository.getVerifiedUsers(pageNumber);
+          final User currentUser = await userRepository.getCurrentUser();
+          yield MembersPageSuccess(
+            users: users,
+            hasReachedMax: false,
+            currentUser: currentUser,
+          );
         }
         if (currentState is MembersPageSuccess) {
-          final users =
-              await userRepository.getVerifiedUsers((currentState.users.length ~/ 10) + 1);
+          final users = await userRepository
+              .getVerifiedUsers((currentState.users.length ~/ 10) + 1);
           yield users.isEmpty
               ? currentState.copyWith(hasReachedMax: true)
               : MembersPageSuccess(
                   users: currentState.users + users,
                   hasReachedMax: false,
+                  currentUser: currentState.currentUser,
```

Comment:
Here while initializing MembersPageSuccess, we need not make the network call again.